### PR TITLE
Add new filters, update UI

### PR DESCRIPTION
**Pull request by simontuffs**
Add  different filters

Add spike eorting

label with metric type

Strip down UI for multi-user work.

hacks for UI mods.

more hacks

merge from master

### CODE REVIEW

**Comment by simontuffs** [comment]
Please feel free to clean up and merge.

File: src/components/Home.js [hunk 1]
```diff
@@ -1303,19 +1326,20 @@ class Home extends Component {
...
                                     this.state.showInfoData.tags.map((tag, index) => {
-                                        tagSearchStrings[index] = JSON.stringify(tag);
+                                        tagSearchStrings[index] = '"' + index + '": ' + JSON.stringify(tag, null, 1);
                                     });
 
                                     //this.setState(update(this.state, { showInfoMetricTagFilter: {$set: text } }));
                                     let matchingTags = [];
                                     for (let tag in tagSearchStrings) {
                                         let searchString = tagSearchStrings[tag];
+                                        console.log(searchString)
```
Comment (pinky00707) [inline src/components/Home.js:1342]: Do we need this?

Comment (simontuffs) [reply src/components/Home.js:1342]: I don't think so. Looks like debug cruft. Did I put it in?

Comment (pinky00707) [reply src/components/Home.js:1342]: Yes it was part of the merge pull. I'll go ahead and remove it with my review changeset. Thanks

**Review by pinky00707 — APPROVED** [review]
LGTM. Go ahead and confirm that the latest changeset works for you.

**Comment by simontuffs** [comment]
Confirmed this looks good on my machine.  Please go ahead and merge it with
master.

**Pull request merged**